# HG changeset patch
# User Rik <rik@octave.org>
# Date 1345668389 25200
#      Wed Aug 22 13:46:29 2012 -0700
# Node ID ae6b7ee0a73394b212d5da9b7bdb1bd90026ba7f
# Parent  336f424066714e4d4a5a4d7f6364d137106f0219
print.m: Correct typo preventing fontsize to be set (bug #37163)

* print.m: Change variable name to from opt to opts.

diff --git a/scripts/plot/print.m b/scripts/plot/print.m
--- a/scripts/plot/print.m
+++ b/scripts/plot/print.m
@@ -396,17 +396,17 @@ function print (varargin)
         set (h(ishandle (h)), "fontname", opts.font);
       endif
       if (! isempty (opts.fontsize))
         if (ischar (opts.fontsize))
           fontsize = str2double (opts.fontsize);
         else
           fontsize = opts.fontsize;
         endif
-        if (! isempty (opts.scalefontsize) && ! opt.scalefontsize != 1)
+        if (! isempty (opts.scalefontsize) && ! opts.scalefontsize != 1)
           ## This is done to work around the bbox being whole numbers.
           fontsize = fontsize * opts.scalefontsize;
         endif
         set (h(ishandle (h)), "fontsize", fontsize);
       endif
     endif
 
     ## call the graphcis toolkit print script
